project/Dependencies.scala

```diff
@@ -10,6 +10,7 @@ object Dependencies {
     val circe              = "0.14.5"
     val zio                = "2.0.9"
     val zioInteropReactive = "2.0.1"
+    val zioJson            = "0.4.2+48-706bdbae-SNAPSHOT"
```

Comment:
Why did you have to use a snapshot here? Why not just go with a regular `0.4.2`?

Reply:
Due to this
```
    def isNull: Boolean        = json.asNull.nonEmpty
    def isArray: Boolean       = json.asArray.nonEmpty
    def isBoolean: Boolean     = json.asBoolean.nonEmpty
    def isString: Boolean      = json.asString.nonEmpty
    def isNumber: Boolean      = json.asNumber.nonEmpty
    def isId: Boolean          = json.asObject.nonEmpty && json.asObject.exists(_.contains(idTag))
    def isDate: Boolean        = json.asObject.nonEmpty && json.asObject.exists(_.contains(dateTag))
 ```
 
 The regular `0.4.2` does not have the `as***` methods and there is no way to check JSON type.
 I've asked them to cut a new release, hopefully they ship `0.4.3` soon so we can depend on stable release.

Reply:
Understood 👍